src/platform/packages/shared/kbn-unified-field-list/src/components/field_item_button/field_item_button.scss

```diff
@@ -74,7 +74,6 @@
 }
 
 .unifiedFieldListItemButton--missing {
-  background: lightOrDarkTheme(transparentize($euiColorMediumShade, .9), $euiColorEmptyShade);
```

Comment:
I removed it completely without any replacement with tokens, as it didn't have any effect at all. Only text color is changing on current main branch.

Reply:
Agree, it does not have an effect currently. Possible, it was necessary in the past for Lens items under "Empty fields" section.